parser/prism/Translator.cc

```diff
@@ -66,6 +66,23 @@ std::unique_ptr<parser::Node> Translator::translate(pm_node_t *node) {
 
             return make_unique<parser::Pair>(parser.translateLocation(loc), std::move(key), std::move(value));
         }
+        case PM_BEGIN_NODE: {
+            auto beginNode = reinterpret_cast<pm_begin_node *>(node);
+            auto loc = &beginNode->base.location;
+
+            NodeVec statements;
+
+            if (beginNode->statements != nullptr) {
+                auto body = beginNode->statements->body;
+                auto prismStmts = absl::MakeSpan(body.nodes, body.size);
+
+                for (auto &prismStmt : prismStmts) {
+                    statements.emplace_back(translate(prismStmt));
+                }
+            }
```

Comment:
I think you can reuse `translateStatements()` here, which handles other "begin-like" things like `()`, `#{ }`, method bodies, etc.:

```suggestion
            auto inlineIfSingle = false;
            auto statements = translateStatements(beginNode->statements, inlineIfSingle);
```